app/view/lib/bolt/obj-stack.js

```diff
@@ -17,7 +17,6 @@ var Stack = Backbone.Model.extend({
         // In the modal dialog, to navigate folders..
         $('#selectImageModal-stack').on('click', '.folder', function (e) {
             e.preventDefault();
-            alert('hoi');
```

Comment:
And nobody noticed that for a long time. Is this as with TODOs. If that nobody notices for half a year the function can be removed? ;-)